cstore_fdw.h

```diff
@@ -170,6 +170,15 @@ typedef struct StripeSkipList
 } StripeSkipList;
 
 
+typedef struct BlockDataBuffers
```

Comment:
1. I think in other places we have used the term _buffer_ for raw data, and not for deserialized data. So I prefer renaming ```BlockDataBuffers``` to ```ColumnBlockData```, and renaming ```StripeData```, ```ColumnData```, and ```ColumnBlockData``` to ```StripeDataBuffers```, ```ColumnDataBuffers```, and ```ColumnBlockDataBuffers```.

2. We put a comment before each struct.